### PR TITLE
Update animals table

### PR DESCRIPTION
**Pull request by OsvaldoBC**
# Hello Codereviewer!

The milestone in this pull request is :

- Use database transactions.

- Modify and delete data in SQL.

- Prepare complex queries that answer analytical questions.

You can see the screenshots in this link [Screenshot](https://docs.google.com/document/d/1AGwPg4jnH2raUEzvRJe2sMdbJdamKWnqqq4MRgRreQA/edit?usp=sharing)

## Thanks!!

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -8,3 +8,68 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
...
+VALUES ('Charmander', 'Feb 8, 2020', 0, false, 11);
+
+INSERT INTO animals (
+        name,
+        date_of_birth,
+        escape_attempts,
+        neutered,
+        weight_kg
+    )
+VALUES ('Plantmon', 'Nov 15, 2021', 2, true, 5.7);
```
Comment (mckent05) [inline data.sql]: - According to the project requirements, the weight here should be `-5.7kg`. Please fix this 🙏

File: data.sql [hunk 2]
```diff
@@ -8,3 +8,68 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
...
+
+INSERT INTO animals (
+        name,
+        date_of_birth,
+        escape_attempts,
+        neutered,
+        weight_kg
+    )
+VALUES ('Squirtle', 'Apr 2, 1993', 3, false, 12.13);
```
Comment (mckent05) [inline data.sql]: - According to the project requirements, the weight here should be `-12.13kg`. Please fix this 🙏

File: data.sql [hunk 3]
```diff
@@ -8,3 +8,68 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
...
+
+/* Project - query and update animals table */
+INSERT INTO animals (
+        name,
+        date_of_birth,
+        escape_attempts,
+        neutered,
+        weight_kg
+    )
+VALUES ('Charmander', 'Feb 8, 2020', 0, false, 11);
```
Comment (mckent05) [inline data.sql]: - According to the project requirements, the weight here should be `-11.0kg`. Please fix this 🙏

File: data.sql [hunk 4]
```diff
@@ -8,3 +8,68 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
...
+VALUES ('Squirtle', 'Apr 2, 1993', 3, false, 12.13);
+
+INSERT INTO animals (
+        name,
+        date_of_birth,
+        escape_attempts,
+        neutered,
+        weight_kg
+    )
+VALUES ('Angemon', 'Jun 12, 2005', 1, true, 45);
```
Comment (mckent05) [inline data.sql]: - According to the project requirements, the weight here should be `-45kg`. Please fix this 🙏

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @OsvaldoBC, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 
- Good job updating and querying the animals table 👏 


## Required Changes ♻️
- Please make the changes suggested in your `data.sql` file, I suggest you run the queries again and update the screenshots in your PR. 👍 👍 
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by franclobo — APPROVED** [review]
Hi @OsvaldoBC, 

# STATUS: APPROVED 😃 

git-merge

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Good use of queries ✔️ 
- Descriptive Pr ✔️ 

## Optional suggestions
- **[OPTIONAL]** Please 🙏🏽 add the screenshots directly to your PR it allows me to open the image and review it otherwise it is too small and I can't see them.
image


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**